portal-api/src/v1/validation/fields/issued-date.js

```diff
@@ -22,38 +19,42 @@ module.exports = (facility, errorList, deal) => {
     manualInclusionNoticeSubmissionDate: minSubmissionDateTimestamp,
   } = deal.details;
 
-  const dealSubmissionDate = formattedTimestamp(dealSubmissionDateTimestamp);
-  const minSubmissionDate = formattedTimestamp(minSubmissionDateTimestamp);
-  const issuedDate = formattedTimestamp(facility.issuedDate);
-  const today = moment();
+  const dealSubmissionDate = new Date(Number.parseInt(deal.details.submissionDate, 10));
+  const minSubmissionDate = new Date(Number.parseInt(deal.details.manualInclusionNoticeSubmissionDate, 10));
+  const issuedDate = new Date(Number.parseInt(facility.issuedDate, 10));
+  const today = new Date();
 
-  let dateOfSubmission = moment().utc().valueOf();
+  let dateOfSubmission = today.valueOf();
 
   if (dealSubmissionDateTimestamp || minSubmissionDateTimestamp) {
     if (submissionType === CONSTANTS.DEAL.SUBMISSION_TYPE.AIN) {
-      dateOfSubmission = dealSubmissionDate;
+      dateOfSubmission = new Date(dealSubmissionDate.valueOf());
     } else if (minSubmissionDateTimestamp) {
-      dateOfSubmission = minSubmissionDate;
+      dateOfSubmission = new Date(minSubmissionDate.valueOf);
     }
   }
 
-  if (dateHasAllValues(issuedDateDay, issuedDateMonth, issuedDateYear)) {
-    if (!moment(issuedDate).isSameOrAfter(dateOfSubmission, 'day')) {
-      const formattedDealSubmissionDate = moment(dateOfSubmission).format('Do MMMM YYYY');
+  const issuedDayHasValues = hasValue(issuedDateDay) && hasValue(issuedDateMonth) && hasValue(issuedDateYear)
+
+  // This is checking d,m,y and then doing logic with timestamp?
+  if (issuedDayHasValues) {
+    if (isBefore(issuedDate, startOfDay(dateOfSubmission))) {
+      const formattedDealSubmissionDate = format(dateOfSubmission, CONSTANTS.DATE_FORMATS.LONG_FORM_DATE);
 
       newErrorList.issuedDate = {
         text: `Issued Date must be on or after ${formattedDealSubmissionDate}`,
         order: orderNumber(newErrorList),
       };
     }
 
-    if (moment(issuedDate).isAfter(today, 'day')) {
+    if (isAfter(issuedDate, endOfDay(today))) {
       newErrorList.issuedDate = {
         text: 'Issued Date must be today or in the past',
         order: orderNumber(newErrorList),
       };
     }
-  } else if (!facility.issuedDate && !dateHasAllValues(issuedDateDay, issuedDateMonth, issuedDateYear)) {
+  }
+  if (!facility.issuedDate && !issuedDayHasValues) {
```

Comment:
Similarly could be possible to have neither if statement satisfied

Reply:
Good flag! Do you think we should turn this into an `||` or shall we leave it and raise a tech debt ticket?

Reply:
https://ukef-dtfs.atlassian.net/browse/DTFS2-7029